cornflow-server/cornflow/tests/unit/test_sign_up.py

```diff
@@ -120,3 +127,214 @@ def test_signup_deactivated(self):
         )
 
         self.assertEqual(response.status_code, 400)
+
+
+class TestSignUpPublic(TestCase):
+    """Test the public signup endpoint (SIGNUP_ACTIVATED=1)"""
```

Comment:
This class does the same as TestSignUp.test_successful_signup(), why do you need both?